src/DonationForms/resources/registrars/templates/fields/Phone.tsx

```diff
@@ -1,15 +1,102 @@
-import InputMask from 'react-input-mask';
-
 import {PhoneProps} from '@givewp/forms/propTypes';
+import {useEffect, useState} from 'react';
+import IntlTelInput from 'intl-tel-input/react';
+import 'intl-tel-input/build/css/intlTelInput.css';
+import InputMask from 'react-input-mask';
 
-export default function Phone({Label, ErrorMessage, placeholder, description, phoneFormat, inputProps}: PhoneProps) {
+/**
+ * @unreleased
+ */
+export default function Phone({
+    Label,
+    ErrorMessage,
+    placeholder,
+    description,
+    phoneFormat,
+    inputProps,
+    fieldError,
+    intlTelInputSettings,
+}: PhoneProps) {
     const FieldDescription = window.givewp.form.templates.layouts.fieldDescription;
+    const {useFormContext} = window.givewp.form.hooks;
+    const {setValue, setError, trigger} = useFormContext();
+
+    const isIntlTelInput = intlTelInputSettings.initialCountry && intlTelInputSettings.utilsScriptUrl;
+    const isFormBuilderPreview = window.location.href.indexOf('about:srcdoc') > -1;
```

Comment:
Just to give you a high level concept, our fields _should_ not have to think about being in the context of the form builder preview.  The builder preview should be treated like the live form.  I realize this is sometimes easier said than done as you had to make a workaround for a preview issue specifically.  

My question for you is can we apply this workaround without having to check if we're in the form builder preview?  Meaning the workaround effectively applied to the input regardless of preview or live mode.

Reply:
Okay cool

- Re: `utilsScript` let's go with that
- Re: fullscreen - can we do the opposite and set fullscreen to false for both?

Reply:
Well, thinking better, the best solution probably would be to keep the `useFullscreenPopup` property set to `false` in all scenarios and use the dropdown in the mobile as well - check the attached screenshot for reference. This way would be possible to fully remove all references to the form builder.

![image](https://github.com/impress-org/givewp/assets/16308864/04a8e26b-016b-4091-8cde-83f37b2ec854)

Reply:
Resolve in 0c0287ee2fc10840f34276a416fc88ca0fe19592